src/applications/pact-act/containers/questions/CheckboxGroup.jsx

```diff
@@ -126,7 +126,9 @@ CheckboxGroup.propTypes = {
   formResponses: PropTypes.object.isRequired,
   h1: PropTypes.string.isRequired,
   responses: PropTypes.arrayOf(PropTypes.string).isRequired,
-  router: PropTypes.object.isRequired,
+  router: PropTypes.shape({
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.